packages/core/strapi/lib/core/app-configuration/index.js

```diff
@@ -35,7 +36,7 @@ module.exports = (dir, initialConfig = {}) => {
 
   const pkgJSON = require(path.resolve(dir, 'package.json'));
 
-  const configDir = path.resolve(dir || process.cwd(), 'config');
+  const configDir = path.resolve(dir, 'config');
```

Comment:
Can you verify removing this doesn't break some of the commands as we use loadConfiguration directly in some of them ?

Reply:
I can let it to be safe